### DISCUSSION TITLE
Can't injection video

### DISCUSSION THREAD

**Discussion by Timitriy** [Genymobile/feedback#58] [Genymotion Desktop] [answered]
Can't inject any video format with 'Media injection' photos can be injected, tell me what the problem is or what video format can be used ?

**Comment by agavignet** [comment]
Hi @Timitriy,

What operating system are you using and which types of video are you trying to inject?

Adrien

**Reply by Timitriy** [reply]
MacBook Air m1 2020 Ventura 13.4 / video farmat -  mov. mp4. webm. avi.

**Reply by Timitriy** [reply]
video can be put in, but the virtual camera itself can't see it
<img width="871" alt="Снимок экрана 2023-06-21 в 18 32 23" src="https://github.com/Genymobile/feedback/assets/94569128/8b083d47-5cbc-43d2-bda2-f4accab0b479">

**Comment by agavignet** [comment, marked as answer]
@Timitriy We are reproducing the issue, after a deep investigation, this issue comes from a heavy third party library on Apple Silicon. We're planning to fix it but we don't have any ETA yet.

Thank you for having spotted the problem.

**Reply by ljk1291** [reply]
@agavignet, More than 2 years later and this still seems to be an issue on macOS. Any chance it can be fixed?

**Reply by agavignet** [reply]
Hi @ljk1291, we haven't forgotten about that issue — just some other priorities have taken over. To answer your question: yes, there's a great chance we'll have a fix in the coming months. I'll keep you posted here as we make progress.

**Reply by CompartamosPE-egb** [reply]
3 years passed still nothing

**Reply by jmaneyrol69** [reply]
@CompartamosPE-egb Sorry for the lack of updates, but we have not been able to find a satisfying solution yet.

**Comment by dthkhang** [comment]
Hi @agavignet , I have same issue in my Macbook Pro M1 16/256 -  Macos Sequoia verison 15.6.1 - please help me
